.github/workflows/broken-linkcheck.yml

```
@@ -0,0 +1,33 @@
+name: Linkcheck
+on:
+    workflow_dispatch: # Enable manual triggering
+    #schedule:
+    # Run monthly on the 1st day of the month
+        #- cron: '0 0 1 * *'
+    pull_request:
+      paths:
+        - ".github/workflows/broken-linkcheck.yml"
+        - "doc/make.py"
+jobs:
+    linkcheck:
+        runs-on: ubuntu-latest
+        defaults:
+          run:
+                shell: bash -el {0}
+
+        steps:
+        - name: Checkout
+          uses: actions/checkout@v4
+          with:
+                fetch-depth: 0
+
+        - name: Set up Conda
+          uses: ./.github/actions/setup-conda
+
+        - name: Build Pandas
+          uses: ./.github/actions/build_pandas
+
+
+        - name: Run linkcheck script
+          working-directory: ./doc
+          run: python make.py linkcheck
```

Instead of dumping everything to the console, we could just forward the broken links

```suggestion
          run: python make.py linkcheck | grep broken
```

Alternatively, we could also store the output in a file and show the broken ones on failure:

```suggestion
          run: python make.py linkcheck | tee linkcheck.txt
          
        - name: Display broken links
          if: failure()
          working-directory: ./doc
          run: grep broken linkcheck.txt
```

---

There are also lot of anchor error and client errors.

---

> failure()

Don't we need ${{}} for failure() syntax ?

---

> Don't we need ${{}} for failure() syntax ?

No, not inside [`if`](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsif)

> When you use expressions in an `if` conditional, you may omit the `${{ }}` expression syntax because GitHub Actions automatically evaluates the `if` conditional as an expression. However, this rule does not apply everywhere.